instrumentation/avaje-jex-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/avaje/jex/v3_0/JexInstrumentation.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.avaje.jex.v3_0;
```

Comment:
According to existing instrumentation directory naming convention, We could probably combine the directories `avaje` and `jex` into `avajejex`. For example, you can refer to https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/vibur-dbcp-11.0/library/src

Reply:
That naming pattern isn't followed everywhere. For example jetty http client instrumentation uses similar package names to this instrumentation.

Reply:
`avaje.jex` seems good since there are other libraries under https://avaje.io/